dolphinscheduler-dao/src/main/resources/sql/upgrade/3.3.0_schema/mysql/dolphinscheduler_ddl.sql

```diff
@@ -16,22 +16,21 @@
 */
 
 DROP TABLE IF EXISTS `t_ds_process_task_lineage`;
-CREATE TABLE `t_ds_process_task_lineage`
-(
-    `id`                           int      NOT NULL AUTO_INCREMENT,
-    `process_definition_code`      bigint   NOT NULL DEFAULT 0,
-    `process_definition_version`   int      NOT NULL DEFAULT 0,
-    `task_definition_code`         bigint   NOT NULL DEFAULT 0,
-    `task_definition_version`      int      NOT NULL DEFAULT 0,
-    `dept_project_code`            bigint   NOT NULL DEFAULT 0 COMMENT 'dependent project code',
-    `dept_process_definition_code` bigint   NOT NULL DEFAULT 0 COMMENT 'dependent process definition code',
-    `dept_task_definition_code`    bigint   NOT NULL DEFAULT 0 COMMENT 'dependent task definition code',
-    `create_time`                  datetime NOT NULL DEFAULT CURRENT_TIMESTAMP COMMENT 'create time',
-    `update_time`                  datetime NOT NULL DEFAULT CURRENT_TIMESTAMP ON UPDATE CURRENT_TIMESTAMP COMMENT 'update time',
-    PRIMARY KEY (`id`),
-    KEY                            `idx_process_code_version` (`process_definition_code`,`process_definition_version`),
-    KEY                            `idx_task_code_version` (`task_definition_code`,`task_definition_version`),
-    KEY                            `idx_dept_code` (`dept_project_code`,`dept_process_definition_code`,`dept_task_definition_code`)
+CREATE TABLE `t_ds_process_task_lineage` (
```

Comment:
So our strategy for database schema is to change only the unreleased version but not the newly released version?

Reply:
> So our strategy for database schema is to change only the unreleased version but not the newly released version?

I'll do that in the next step in #16525 .